### PR TITLE
Add oVirt modules notes to CHANGELOG

### PR DESCRIPTION
**Pull request by machacekondra**
##### ISSUE TYPE

 - Docs Pull Request

##### COMPONENT NAME

CHANGELOG.md

##### ANSIBLE VERSION

```
2.3.0
```

##### SUMMARY

This patch specify new features of oVirt in CHANGELOG

### CODE REVIEW

**Comment by machacekondra** [comment]
@mwperina

File: CHANGELOG.md [hunk 1]
```diff
@@ -27,6 +27,14 @@ Ansible Changes By Release
   the old overwriting behaviour via the config option.  In 2.5, multiple --tags
   options will be merged with no way to go back to the old behaviour.
 
+###Modules Notes:
```
Comment (gundalow) [inline CHANGELOG.md:30]: I like the idea of "Module Notes:` though I wonder if it should go *after* the list of new modules

File: CHANGELOG.md [hunk 2]
```diff
@@ -27,6 +27,14 @@ Ansible Changes By Release
...
+- oVirt
+  * Add support for oVirt 4.1 features.
+  * Add support for data centers, clusters, hosts, storages and networks management.
```
Comment (gundalow) [inline CHANGELOG.md]: "storages" doesn't seem right, 
storage locations, storage types?

**Review by mwperina — COMMENTED** [review]
We would like emphasize changes in oVirt modules, so that's we have added a new section "Modules Notes". Just listing new modules doesn't provide any higher level overview of changes.

**Review by gundalow — CHANGES_REQUESTED** [review]
needs_revision

File: CHANGELOG.md [hunk 3]
```diff
@@ -27,6 +27,14 @@ Ansible Changes By Release
   the old overwriting behaviour via the config option.  In 2.5, multiple --tags
   options will be merged with no way to go back to the old behaviour.
 
+###Modules Notes:
+- oVirt
+  * Add support for oVirt 4.1 features.
```
Comment (mykaul) [inline CHANGELOG.md]: oVirt -> oVirt/RHV

File: CHANGELOG.md (hunk 2 above)
Comment (mykaul) [reply CHANGELOG.md]: Indeed - storage domains is the correct term.

**Review by mykaul — COMMENTED** [review]
Did inventory get in for Ansible 2.2, or is it part of the work for 2.3?

File: CHANGELOG.md (hunk 1 above)
Comment (mwperina) [reply CHANGELOG.md:30]: I'm not sure if most users don't stop reading at potentially huge list of new modules and will ever get to "Modules notes" section if it's after "New modules" section

Comment (gundalow) [reply CHANGELOG.md:30]: That's a fair point. You can drop my issue.

File: CHANGELOG.md [hunk 4]
```diff
@@ -27,6 +27,15 @@ Ansible Changes By Release
   the old overwriting behaviour via the config option.  In 2.5, multiple --tags
   options will be merged with no way to go back to the old behaviour.
 
+###Modules Notes:
+- oVirt/RHV
+  * Add dynamic inventory.
```
Comment (mwperina) [inline CHANGELOG.md:32]: I'd change to "Add dynamic inventories for oVirt resources"

**Comment by gundalow** [comment]
If once you are app happy please reply with "shipit" and I'll merge this

File: CHANGELOG.md (hunk 4 above)
Comment (machacekondra) [reply CHANGELOG.md:32]: It's just one and for VMs. What you mean are Anisble facts modules for most of the resources.

Comment (mwperina) [reply CHANGELOG.md:32]: Ahh, OK, thanks for explanation

**Comment by mwperina** [comment]
shipit

**Review by gundalow — APPROVED** [review]

**Comment by gundalow** [comment]
Merged, thanks

**Pull request merged**